src/VoidHandler.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * A dummy handler that returns undefined.
+ */
+export default class VoidHandler {
+  async execute() {
+    return undefined;
```

Comment:
The `async` seems wrong. This will return a Promise to undefined?
Also, why just not an empty method?